### DISCUSSION TITLE
Remote cluster keeps returning after removal

### DISCUSSION THREAD

**Discussion by THExCAPNxRK** [Security-Onion-Solutions/securityonion#4086] [Unsupported Versions] [answered]
I am trying to remove an old cluster from elastic search and it keeps coming back causing searches to take forever. Following the documentation I am running the following in DevTools in kibana.

PUT _cluster/settings
{
  "persistent": {
    "search": {
      "remote": {
        "so-commercial-sensor_heavynode": {
          "skip_unavailable": null,
          "seeds":null
        }
      }
    }
  }
}

It runs successfully and the node disappears from the stack management page for a little bit.

Here is the out of "curl -sk https://localhost:9200/_cluster/settings"

{"persistent":{"cluster":{"remote":{"so-commercial-sensor_heavynode":{"skip_unavailable":"true","seeds":["so-commercial-sensor:9300"]}

I am running SO version 2.3.40

Any help on this is appreciated.

**Comment by TOoSmOotH** [comment, marked as answer]
You will need to remove it from the nodes tab pillar.
